sites/public/pages/_app.tsx

```diff
@@ -1,5 +1,6 @@
 import "@bloom-housing/ui-components/src/global/index.scss"
 import "../styles/overrides.scss"
+import "./eligibility/age.scss"
```

Comment:
Seems like we shouldn't be importing the eligibility styles into the root of the app?

Reply:
See comment below